### PR TITLE
Update Vercel Integration Doc

### PR DESCRIPTION
**Pull request by zhangyangyu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by xuanyu66 — APPROVED** [review]

File: tidb-cloud/integrate-tidbcloud-with-vercel.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ The detailed steps are as follows:
 5. On the left side, select the target Vercel projects and framework your Vercel projects using. If the framework isn't listed, select **General**. Different frameworks determine different environment variables.
 6. On the right side, select the target TiDB Cloud cluster after providing the cluster information. Each TiDB Cloud cluster belongs to [an organization and a project](/tidb-cloud/manage-user-access.md#organizations-and-projects).
 7. Click **Add Integration and Return to Vercel**.
+
+![Vercel Integration Page](/media/tidb-cloud/integration-vercel-link-page.png)
```
Comment (ran-huang) [inline tidb-cloud/integrate-tidbcloud-with-vercel.md:58]:
```suggestion
    ![Vercel Integration Page](/media/tidb-cloud/integration-vercel-link-page.png)
```

File: tidb-cloud/integrate-tidbcloud-with-vercel.md [hunk 2]
```diff
@@ -83,35 +86,26 @@ DATABASE_URL
...
+datasource db {
+    provider = "mysql"
+    url      = env("DATABASE_URL")
+}
 ```
 
-In Vercel, you can declare the variables as follows. You can customize the name according to your project need.
+In Vercel, you can declare the environment variables as follows.
+
+- **Key** = DATABASE_URL
```
Comment (ran-huang) [inline tidb-cloud/integrate-tidbcloud-with-vercel.md:105]:
```suggestion
- **Key** = `DATABASE_URL`
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

**Pull request merged**